src/Smp.Web/ClientApp/src/app/models/conversation.ts

```diff
@@ -0,0 +1,12 @@
+import { FriendlyMessage } from './message';
+import { User } from './user';
+
+export class Conversation {
+  public id: string;
+  public createdAt: Date;
+}
+
+export class ExtendedConversation extends Conversation {
```

Comment:
Not sure if there's a better way of doing this that we can adapt. Most of the models are basically copies of the C# models, but we frequently find ourselves needing more information to make the user experience better, so I added an extended version (perhaps should've matched the "Friendly" use with messages) to allow us to put this information somewhere to be used in the template.